plugins/element-hider-luna/src/index.ts

```
@@ -0,0 +1,609 @@
+		elementObserver = null;
+	}
+	contextMenuObserver.disconnect();
+	
+	// Close any open custom menu
+	closeCustomMenu();
+	
+	// Remove highlights
+	removeHighlight();
+	
+	// Clean up global functions
+	delete (window as any).showAllElementsFromSettings;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)